packages/hooks/package.json

```diff
@@ -17,16 +17,64 @@
     "type": "git",
     "url": "git+https://github.com/Developer-DAO/web3-ui/"
   },
-  "main": "index.js",
   "scripts": {
-    "types": "index.d.ts",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "yarn clean; rollup -c",
+    "clean": "rimraf dist",
+    "format": "prettier --write \"src/**/*.{ts,tsx,json,js,jsx}\"",
+    "format:check": "prettier --list-different \"src/**/*.{ts,tsx,json,js,jsx}\"",
+    "test": "jest --maxWorkers=2",
+    "test:watch": "yarn test --watch",
+    "test:coverage": "jest --coverage --colors --maxWorkers=2",
+    "hygen": "hygen",
+    "component:new": "hygen component with-prompt",
```

Comment:
do we need this? it looks specific to components

Reply:
Probably not. I'll remove it